x-pack/plugins/security_solution/public/resolver/view/assets.tsx

```diff
@@ -136,17 +72,70 @@ const PaintServers = memo(() => (
       <stop offset="1" stopColor="#54b399" />
     </linearGradient>
     <linearGradient
-      id={PaintServerIds.terminatedTriggerCube}
-      x1="-381.18658"
-      y1="264.68187"
-      x2="-380.48546"
-      y2="263.8817"
+      id={PaintServerIds.runningTriggerCube}
+      x1="-381.18643"
+      y1="264.68195"
+      x2="-380.48514"
+      y2="263.8816"
       gradientTransform="matrix(70.05179, 0, 0, -79.94774, 26724.01618, 21181.09848)"
       gradientUnits="userSpaceOnUse"
     >
       <stop offset="0" stopColor="#dd0a73" />
       <stop offset="1" stopColor="#f66" />
     </linearGradient>
+    {isDarkMode ? (
+      <>
+        <linearGradient
+          id={PaintServerIds.terminatedProcessCube}
+          x1="-381.23752"
+          y1="264.24026"
+          x2="-380.48514"
+          y2="263.3816"
+          gradientTransform="matrix(70.05178, 0, 0, -79.94771, 26724.01313, 21140.72096)"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop offset="0" stopColor="#4c82c3" />
+          <stop offset="1" stopColor="#8bd1c7" />
+        </linearGradient>
+        <linearGradient
+          id={PaintServerIds.terminatedTriggerCube}
+          x1="-381.18658"
+          y1="264.68187"
+          x2="-380.48546"
+          y2="263.8817"
+          gradientTransform="matrix(70.05179, 0, 0, -79.94774, 26724.01618, 21181.09848)"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop offset="0" stopColor="#dd0a73" />
+          <stop offset="1" stopColor="#f66" />
+        </linearGradient>
+      </>
+    ) : (
+      <>
+        <linearGradient
+          id={PaintServerIds.terminatedProcessCube}
+          x1="10.5206"
```

Comment:
:grey_question: just curious - why are we using different US offsets for the light mode gradients?

Reply:
I thought the same thing, but just following the exported SVG I was provided 🤔